audio/playtone/source/main.c

```diff
@@ -19,6 +19,29 @@ void fill_audio_buffer(void* audio_buffer, size_t offset, size_t size, int frequ
     }
 }
 
+void audout_play(Handle event, AudioOutBuffer source_buffer, AudioOutBuffer released_buffer) {
```

Comment:
Shouldn't this be passed by pointer instead of by value? Currently this is making copies of the struct everytime the function is called...

Reply:
Of course